compiler/noirc_evaluator/src/ssa/plonky2_gen/mod.rs

```diff
@@ -760,6 +771,12 @@ impl Builder {
                             return Err(Plonky2GenError::UnsupportedFeature { name: feature_name });
                         }
                     },
+                    Value::Function(function_id) => {
+                        if !self.is_function_call_safe_to_ignore(function_id) {
+                            let feature_name = format!("calling {:?}", func);
```

Comment:
Please, introduce some difference from the next error message, to make it easier to discern between the two if one of them manifests during execution.

Maybe "calling function {:?}" here, but up to you.

Reply:
Done: https://github.com/blocksense-network/noir/pull/48/commits/efb3949d89d2fd1284335a2ba12c5f7dee3f6299